guides/release/components/helper-functions.md

```diff
@@ -73,7 +72,7 @@ To implement the helper, we write a JavaScript function that takes its arguments
 arguments, which we'll discuss next.
 
 ```js {data-filename="app/helpers/substring.js"}
-import { helper } from "@ember/component/helper";
+import { helper } from '@ember/component/helper';
```

Comment:
@jenweber @locks should we single or double quote as standard in .js / .hbs? just checking before merging

lgtm otherwise!

Reply:
I think the default linting rule is single quotes for .js and double quotes for .hbs, so this change should be good!

Reply:
thank you!